crates/wasmtime/src/instance.rs

```diff
@@ -179,6 +195,19 @@ impl Instance {
         })
     }
 
+    pub fn new(store: &Store, module: &Module, imports: &[Extern]) -> Result<Instance, Error> {
+        return _new(store, module, imports, None);
+    }
+
+    pub fn new_with_user_state(
+        store: &Store,
+        module: &Module,
+        imports: &[Extern],
+        user_state: Option<dyn Any>,
+    ) -> Result<Instance, Error> {
+        return _new(store, module, imports, user_state);
+    }
```

Comment:
```suggestion
    pub fn new(store: &Store, module: &Module, imports: &[Extern]) -> Result<Instance, Error> {
        Instance::new_with_user_state(store, module, imports, None)
    }
```

This function is also missing documentation.

Reply:
Fixed, `_new` is removed.

One can access the user state by `instance.host_state()` then cast it to `HostState`